notes/decentralization-autonomy-organisation-brekke.md

```diff
@@ -0,0 +1,76 @@
+# Overview 
```

Comment:
I think you can skip the overview/preamble and just have the main heading (or if you want to keep that move to bottom).